launcher/src/backend/ethereum-services/TekuBeaconService.int.js

```diff
@@ -15,7 +17,7 @@ test('teku validator import', async () => {
         name: 'Teku--integration-test--ubuntu-2204',
         image: 'ubuntu-22.04',
         location: 'fsn1',
-        server_type: 'cpx21',
+        server_type: 'cx31',
```

Comment:
is it necessary to upgrade VM? to run Teku

Reply:
ups no hetzner just had issues `Error: resources not available` and then i had to switch server type 😅

Reply:
got it 👍